rules/app.bzl

```diff
@@ -10,6 +10,7 @@ _IOS_APPLICATION_KWARGS = [
     "test_host",
     "families",
     "entitlements",
+    "entitlements_validation",
```

Comment:
Do you still want to set the default value if it's not set?

Reply:
the default won't be overridden if nothing is passed in